Lib/_pydecimal.py

```diff
@@ -2915,18 +2933,19 @@ def _iseven(self):
         return self._int[-1+self._exp] in '02468'
 
     def adjusted(self):
-        """Return the adjusted exponent of self"""
+        """Return the adjusted exponent of the number.\n\
+        Defined as exp + digits - 1.
+        """
         try:
             return self._exp + len(self._int) - 1
         # If NaN or Infinity, self._exp is string
         except TypeError:
             return 0
 
     def canonical(self):
-        """Returns the same Decimal object.
-
-        As we do not have different encodings for the same number, the
-        received object already is in its canonical form.
+        """Return the canonical encoding of the argument. Currently, the encoding
+        of a Decimal instance is always canonical, so this operation returns its
+        argument unchanged.
```

Comment:
It is recommended to keep the docstring in the form:
```
<summary (one line)>
<empty line>
<more details>
```
`pydoc` has some preferences for this. Argument Clinic requires this format.

Please insert an empty string in both versions if it was in the Python version.